pages/employment/index.tsx

```diff
@@ -90,27 +97,35 @@ export default function EmploymentPage(
           <a id="more" />
           <div className="w-full bg-benefits bg-cover bg-fixed bg-center bg-no-repeat py-12">
             <div className="mx-auto max-w-9xl px-4">
-              <TinaMarkdown
-                components={componentRenderer}
-                content={data.employment._body}
-              />
-              <Benefits data={data.employment.benefits} />
-              {data.employment.benefitsBody && (
+              <div data-tina-field={tinaField(data.employment, "_body")}>
                 <TinaMarkdown
-                  content={data.employment.benefitsBody}
                   components={componentRenderer}
+                  content={data.employment._body}
                 />
+              </div>
+              <Benefits data={data.employment.benefits} />
+              {data.employment.benefitsBody && (
+                <div
+                  data-tina-field={tinaField(data.employment, "benefitsBody")}
+                >
+                  <TinaMarkdown
+                    content={data.employment.benefitsBody}
+                    components={componentRenderer}
+                  />
+                </div>
               )}
             </div>
           </div>
         </Section>
         {data.employment.afterBody ? (
           <Section className="prose-employment mb-16 text-center">
             <Container padding="px-4" className="w-full">
-              <TinaMarkdown
-                content={data.employment.afterBody}
-                components={componentRenderer}
-              />
+              <div data-tina-field={tinaField(data.employment, "afterBody")}>
+                <TinaMarkdown
+                  content={data.employment.afterBody}
+                  components={componentRenderer}
+                />
+              </div>
```

Comment:
cc @amankumarrr 
Why cant we put the `data-tina-field` attribute on `<TinaMarkdown/>`?

```suggestion
                <TinaMarkdown
                  content={data.employment.afterBody}
                  components={componentRenderer}
                  data-tina-field={tinaField(data.employment, "afterBody")}
                />
```

Reply:
@bradystroud data-tina-field is an HTML property, we can't place it directly on any markdown or component.